### ISSUE TITLE
Suggest using Must instead of Chai

### ISSUE THREAD

**Issue by nwinch** [Prismatik/babel-node-starter#1]
Could we use Must.js in place of Chai.js?  We've used it on quite a few projects with good results.  It'd be nice to continue using it for our boilerplate projects.

**Comment by Nicktho** [comment]
I'm going to put forward an alternative that we remove the assertion library all together, `npm install`ing an assertion library is a trivial task, I believe we should only have libraries that need some kind of configuration in the boilerplate.

**Comment by nwinch** [comment]
Yeah that's fair.  An assertion lib should probably be dealer's choice.
